addons/website_mass_mailing/i18n/website_mass_mailing.pot

```diff
@@ -137,13 +137,20 @@ msgstr ""
 msgid "Form Subscription"
 msgstr ""
 
+#. module: website_mass_mailing
+#. odoo-python
+#: code:addons/website_mass_mailing/controllers/website_form.py:0
+#, python-format
+msgid "Mailing List(s) not found!"
```

Comment:
Hello @paso-odoo , can you not modify anything other than the translation terms of your fix